clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -767,8 +767,13 @@ Expected<StringRef> wrapSYCLBinariesFromFile(StringRef InputFile,
         inconvertibleErrorCode(),
         "can't wrap SYCL image. -triple argument is missed.");
 
+  // SYCL runtime currently works for spir64 target triple and not for
```

Comment:
This change is not directly in the scope of this PR. But it fixes an issue that was exposed by running E2E SYCL test.

Thanks